map2loop/thickness_calculator.py

```diff
@@ -261,9 +260,18 @@ def compute(
         contacts = contacts.sjoin(basal_contacts, how="inner", predicate="intersects")
         contacts = contacts[["X", "Y", "Z", "geometry", "basal_unit"]].copy()
         bounding_box = map_data.get_bounding_box()
+        
         # Interpolate the dip of the contacts
         interpolator = DipDipDirectionInterpolator(data_type="dip")
-        dip = interpolator(bounding_box, structure_data, interpolator=scipy.interpolate.Rbf)
+        
+        ### TODO: RC - fix the Linalg error at some stage. 
+        # for now, if error is raised, return thicknesses with -1
+        try:
```

Comment:
Is the linalg error coming from colocated points?

Reply:
the collocated points error has been fixed in the meantime. Removing this code.